### PR TITLE
add GetExAbsCurrentMC command

### PR DESCRIPTION
**Pull request by DarkPark**

### CODE REVIEW

File: docs/commands/GetCurrentMC.md [hunk 1]
```diff
@@ -64,39 +64,39 @@ It's pulse counter's values for selected channels.
...
-| channel #1 value | `8`   |                                                                         | `08`     |
-| channel #2 value | `10`  |                                                                         | `0a`     |
-| channel #3 value | `12`  |                                                                         | `0c`     |
+| Field             | Value              | Bits                                                                    | Hex      |
+| ----------------- | ------------------ | ----------------------------------------------------------------------- | -------- |
+| command id        | `24`               |                                                                         | `0x18`   |
+| command size      | `6`                |                                                                         | `0x06`   |
+| channels          | `1`, `2`, `3`, `4` | `0b00001111`                                                            | `0x0f`   |
+| channel `1` value | `131`              | `0b0000000010000011` <br> with extended bits: <br> `0b0000000110000011` | `0x8301` |
```
Comment (DeadByElpy) [inline docs/commands/GetCurrentMC.md:72]: does it good to make an example with real INT bit to explain EXTENDED value? Will be good example at that special type, here - these integer values don't exists in real

File: docs/commands/GetExAbsCurrentMC.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+[current](../parameter-types.md#data-type) i.e. = `2`.
+
+
+## Request
+
+### Format
+
+| Size | Type   | Field               |
+| ---- | ------ | ------------------- |
+| `1`  | `byte` | extra flag = `0x1f` |
```
Comment (DeadByElpy) [inline docs/commands/GetExAbsCurrentMC.md:15]: extra or extended header flag? just ask for make it identically everywhere

File: docs/commands/uplink/ExAbsDayMC.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+[See details](../../types.md#channels-bit-set).
+
+#### **channel pulse coefficient**
+
+This field is the device's pulse coefficient that determines the correspondence of the consumed resource to 1 pulse.
+
+### Examples
+
+#### channels `1` for `2023.03.10 00:00:00 GMT`:
```
Comment (DeadByElpy) [inline docs/commands/uplink/ExAbsDayMC.md:48]:
```suggestion
#### channel `1` for `2023.03.10 00:00:00 GMT`:
```
?

File: docs/types.md [hunk 4]
```diff
@@ -234,7 +235,7 @@ Usually this bit set takes from `1` to `5` bytes as there are 4 channels max ava
 
 One byte `0b00001111` which is the same with extended bits or `0x0f`.
 
-#### channels #5, #6, #12:
+#### channels `6`, `7`, `13`:
 
 Two bytes `0b0001000001100000`. It becomes `0b0010000011100000` with extended bits or `0xe020`.
```
Comment (DeadByElpy) [inline docs/types.md:240]: Two bytes becomes two bytes, one example enough

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: docs/types.md (hunk 4 above)
Comment (DarkPark) [reply docs/types.md:240]: they are different

File: docs/commands/uplink/ExAbsDayMC.md (hunk 3 above)
Comment (DarkPark) [reply docs/commands/uplink/ExAbsDayMC.md:48]: fixed

File: docs/commands/GetExAbsCurrentMC.md (hunk 2 above)
Comment (DarkPark) [reply docs/commands/GetExAbsCurrentMC.md:15]: it's `extra flag` everywhere

but we can discuss a new name

File: docs/commands/GetCurrentMC.md (hunk 1 above)
Comment (DarkPark) [reply docs/commands/GetCurrentMC.md:72]: why they don't exist? `131` is very real value of the first channel

File: docs/commands/GetExAbsCurrentMC.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
+# GetExAbsCurrentMC
+
+Command to request/receive current consumption absolute values from device channels.
```
[the remark this answers is not in the document]
Comment (DarkPark) [reply docs/commands/GetExAbsCurrentMC.md:3]: let's discuss

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**